core/src/parser/utils.rs

```diff
@@ -845,18 +845,8 @@ pub fn strip_indent(mut chunks: Vec<StrChunk<RichTerm>>) -> Vec<StrChunk<RichTer
     chunks
 }
 
-/// Strip the indentation of a doc metadata. Wrap it as a literal string chunk and call
-/// [`strip_indent`].
-pub fn strip_indent_doc(doc: String) -> String {
-    let chunk = vec![StrChunk::Literal(doc)];
-    strip_indent(chunk)
-        .into_iter()
-        .map(|chunk| match chunk {
-            StrChunk::Literal(s) => s,
-            _ => panic!("expected literal string after indentation of documentation"),
-        })
-        .next()
-        .expect("expected non-empty chunks after indentation of documentation")
+pub fn mk_strchunk_literal<E>(s: impl AsRef<str>) -> StrChunk<E> {
```

Comment:
Should this be a function of `StrLiteral` instead? `StrChunk::literal` or something more explicit if we want to avoid confusion around the fact that this does some normalization, like `normalize_literal` or whatnot?

Reply:
That sounds like a good idea 👍